tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "ES2022",
-    "module": "ES6",
+    "module": "NodeNext",
```

Comment:
Was this required? 🤔

Reply:
Yes, it gives an error by the TypeScript compiler.

```
tsconfig.json:4:15 - error TS5110: Option 'module' must be set to 'NodeNext' when option 'moduleResolution' is set to 'NodeNext'.
4     "module": "ES6",
```